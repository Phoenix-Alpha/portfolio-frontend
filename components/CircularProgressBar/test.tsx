import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { CircularProgressBar } from '.'

describe('<CircularProgressBar />', () => {
  it('should renders correct <CircularProgressBar />', () => {
    const wrapper = shallow(<CircularProgressBar progress={0} />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
