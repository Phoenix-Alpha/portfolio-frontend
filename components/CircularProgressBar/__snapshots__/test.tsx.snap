// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CircularProgressBar /> should renders correct <CircularProgressBar /> 1`] = `
<svg
  height="100"
  viewBox="-25 -25 400 400"
  width="100"
>
  <circle
    cx="175"
    cy="175"
    fill="none"
    r="175"
    stroke="#558FFF"
    strokeWidth="25"
  />
  <circle
    cx="175"
    cy="175"
    fill="none"
    r="175"
    stroke="#558FFF"
    strokeDasharray="1100"
    strokeDashoffset="1100"
    strokeLinecap="butt"
    strokeWidth="25"
    style={
      Object {
        "strokeDashoffset": 1100,
        "transition": "stroke-dashoffset 1s ease-out",
      }
    }
    transform="rotate(-90 175 175)"
  />
  <text
    dominantBaseline="central"
    fill="#FFFFFF"
    style={
      Object {
        "fontFamily": "inherit",
        "fontSize": 48,
        "fontWeight": "bold",
      }
    }
    textAnchor="middle"
    x={175}
    y={175}
  >
    0
    <tspan
      dx={10}
    >
      %
    </tspan>
  </text>
</svg>
`;
